backend/vkt/src/main/resources/oph-configuration/vkt.properties.template

```diff
@@ -15,3 +15,5 @@ email.sending-enabled=true
 email.service-url=${virkailija.host.alb}/ryhmasahkoposti-service/email/firewall
 
 reservation.duration=PT30M
+
+cas-oppija.validate-ticket-url=https://testiopintopolku.fi/cas-oppija/serviceValidate
```

Comment:
Tänne voisi vaihtaa `cas-oppija.validate-ticket-url={{opintopolku_baseurl}}/cas-oppija/serviceValidate` niin toimisi eri ympäristöissä.

Vaihtoehtoisesti voisi käyttää avainta `cas_oppija_suomifi_service_provider_entity_id`, joka esim. pallerolla on `https://testiopintopolku.fi/cas-oppija`, mut ehkä menisin itse tuolla ylemmällä jossa määriteltäis url tarkemmin itse. Tiedä missä käytössä tä alempi lopulta on. Kaivoin nämä siis cloud-environment repojen opintopolku.yamlista.

Reply:
Lisätään tänne alle myös tuo `auth.service-url` joka vastaa tuota tunnistautumisurlia frontendissä. Oliskohan se hyvä olla cloud-environment repoissa ihan omana avaimenaan esim. nimellä `vkt_auth_url` ? Jos muuten tuo auth.service-url on vaan tuo frontin tunnistaumissivun url niin ehkä sen vois jotenkin uudelleennimetä?

Reply:
Joo, tuo on vain tuo frontin tunnistautumissivun url. Toi auth on irrallinen nyt tässä kun toi service-url on täysin cas-oppijan termistöä niin parempi paikka sille `cas-oppija.service-url`

Reply:
Tulipa mieleen että tän voisi toteuttaa myös vaikka ympäristömuuttujan `vkt_app_url: https://vkt.untuvaopintopolku.fi` ja sama pallerolla ja sade-domainille tms. kautta. Jos päätyy hyödyntämään tuota myöhemminkin jossain. Mut sinänsä kai ihan kummin vaan.